### DISCUSSION TITLE
News for mongodb support?

### DISCUSSION THREAD

**Discussion by sawa-ko** [prisma/prisma#8994] [Q&A] [answered]
I would love to use mongodb with prisma properly, as the current implementation is like using mongodb as a relational database, when mongodb is an object based database.

What's new? I see that some own feature for mongodb has not been implemented for some time.

**Comment by ryands17** [comment, marked as answer]
@kaname-png 👋 

Currently MongoDB is in preview so there are some [limitations](https://www.prisma.io/docs/concepts/database-connectors/mongodb#known-limitations). We have feature requests for making support for Mongo better and you can follow those for more updates 🙂 

https://github.com/prisma/prisma/issues/6708

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
